pkg/serverless/trace/trace.go

```diff
@@ -56,17 +56,15 @@ func (l *LoadConfig) Load() (*config.AgentConfig, error) {
 // Start starts the agent
 func (s *ServerlessTraceAgent) Start(enabled bool, loadConfig Load) {
 	if enabled {
-		// during hostname resolution the first step is to make a GRPC call which is timeboxed to a 2 seconds deadline
-		// in the serverless mode, we don't start the GRPC server so this call will fail and cause a 2 seconds delay
-		// by setting cmd_port to -1, this will cause the GRPC client to fail instantly
-		ddConfig.Datadog.Set("cmd_port", "-1")
+		// setting the hostname here saves about 1s in load time
+		ddConfig.Datadog.Set("hostname", "lambda")
 
 		tc, confErr := loadConfig.Load()
+		tc.Hostname = ""
```

Comment:
We must set the hostname to empty string in all cases, not just when there is no error.